packages/server/postgres/migrations/1659973034251_alterMeetingSeries.ts

```diff
@@ -0,0 +1,26 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DELETE FROM "MeetingSeries"; --Intentionally without 'WHERE' clause.
+
+    ALTER TABLE "MeetingSeries"
+      ADD COLUMN IF NOT EXISTS "teamId" VARCHAR(100) NOT NULL,
+      ADD COLUMN IF NOT EXISTS "facilitatorId" VARCHAR(100) NOT NULL;
```

Comment:
These columns need to be non-null for the meeting series to be usable, and it's non-trivial to set these with data on the rethink meetings.

Since this is a WIP feature, I've decided to simply delete all rows currently in any meeting series table, instead of investing time into making this migration "clean", but I'm very open to ways to improve this.

Reply:
Recurrence is behind the feature flag, I'd rather delete some rows by hand (if any) than have hard delete in the mutation 💀 (I checked, both staging and prod are empty)

Reply:
Fair enough! Removed the deletion